src/components/charts/DailyChart.vue

```diff
@@ -84,12 +84,13 @@ export default class DailyChart extends Vue {
           show: false,
           pieces: [
             {
-              max: -0.01,
-              min: this.absoluteMin - 2,
+              max: 0.0,
+              min: -2,
               color: 'red',
             },
             {
-              min: -0.00001,
+              min: 0.0,
+              max: 2,
```

Comment:
well you actually did change these / add these ...

Reply:
True. I'll test now with the high limits removed. The TradesLog already didn't have those anyway.

Reply:
the problem is, for this to actually show up as a problem you got to have some very high winning days (which honestly, i had in that test-database accidentally - along with some high losing days) :laughing:

Reply:
Yeah I noticed. It was pretty ludicrous to get 200% 😄 . I noticed removing the limits draws the lines but a thin solid black. I'll test using the absolute min/max functions in an intuitive manner as limits.

Reply:
i think using "min" for the red, and not using max should work.
It's problably an oddity of echarts that's causing one to need both limits and not the other - but could be worse :laughing:

Reply:
Does the latest commit work for you? I noticed it had both min and max functions anyway so it made some sense and it still works here. I didn't see the same for the hourly though and I couldn't find a fast way to reproduce them for it.

Reply:
i think this should work.
I don't expect anyone to have an average for one hour of above 200% :laughing: for one day however, a few longer trades can cause that ...